README.md

```diff
@@ -28,6 +36,11 @@ bundle exec middleman
 
 Then visit [http://localhost:4567/](http://localhost:4567/).
 
+### Spellchecking
+
+If you have a false hit during spellchecking, you can add the word to /data/spelling-exceptions.txt
```

Comment:
Add a period to the end of this line. Perhaps italicize or backtick the filename path.